lemur/notifications/views.py

```diff
@@ -113,6 +114,7 @@ def get(self):
         return service.render(args)
 
     @validate_schema(notification_input_schema, notification_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this resource is actually the most concerning to me that we don't have any access controls on. But, even for this case we'd likely want to start with a default false config property ADMIN_ONLY_NOTIFICATION_WRITE with the eventual goal of defaulting to True

Reply:
another alternative that allows for self service management of notifications while preventing some of the attacks I'm concerned about is to introduce the concept of notification ownership (similar to what we have for roles, authorities, and certificates today). but this is likely to be a somewhat involved change